xonsh/proc.py

```diff
@@ -220,8 +220,8 @@ def __init__(self, fd, timeout=None):
 
 def populate_buffer(reader, fd, buffer, chunksize):
     """Reads bytes from the file descriptor and copies them into a buffer.
-    The reads happend in parallel, using pread(), and is thus only
-    availabe on posix. If the read fails for any reason, the reader is
+    The reads happened in parallel, using pread(), and is thus only
```

Comment:
This sentence confuses me.  Maybe this should be something like the following, instead?

"The reads happen in parallel using the pread() syscall, which is only available on POSIX systems."